tests/integration.rs

```diff
@@ -355,7 +355,7 @@ mod tests {
 
             let mut fw_path = workload_path;
             #[cfg(target_arch = "aarch64")]
-            fw_path.push("Image");
+            fw_path.push("CLOUDHV_EFI.fd");
```

Comment:
Wait this is a big change, you've moving all `aarch64` tests from direct kernel boot to EFI firmware boot. Is this intentional?

Reply:
I intended, but it doesn't impact much on aarch64. Because in most aarch64 test cases, kernel was set with `direct_kernel_boot_path()`.

Reply:
It was changed back.